phoenix-core/src/main/java/org/apache/phoenix/coprocessor/IndexRebuildRegionScanner.java

```diff
@@ -594,14 +376,16 @@ private boolean checkIndexRow(final byte[] indexRowKey, final Put put) throws IO
         return true;
     }
 
-    private void logToIndexToolOutputTable(byte[] dataRowKey, byte[] indexRowKey, long dataRowTs, long indexRowTs,
+    @VisibleForTesting
```

Comment:
Do we still need this annotation if it is a public function?

Reply:
Nope!